UPGRADE-3.0.md

```diff
@@ -15,3 +15,10 @@
 - The signature of the `Event::getId()` method changed to return an instance of the `EventId` class instead of a `string`
 - The signature of the `Options::getDsn()` method changed to always return an instance of the `Dsn` class instead of a `string`
 - Removed the `Options::getProjectId`, `Options::getPublicKey` and `Options::getSecretKey` methods, use `Options::getDsn()` instead.
+- Removed the `Breadcrumb::LEVEL_CRITICAL` constant. Use `Breadcrumb::LEVEL_FATAL` instead
+- Removed the `Breadcrumb::levelFromErrorException()` method
+- Removedc the `PluggableHttpClientFactory` class
+- Removed the `ClientBuilderInterface::setUriFactory()`, `ClientBuilderInterface::setMessageFactory()`, `ClientBuilderInterface::setTransport()`, `ClientBuilderInterface::setHttpClient()`, `ClientBuilderInterface::addHttpClientPlugin()` and `ClientBuilderInterface::removeHttpClientPlugin()` methods. Use `ClientBuilderInterface::setTransportFactory()` instead
```

Comment:
Can we rephrase this (and the others) with multiple methods like this? It's should be easier to scan:
```suggestion
- Removed the following methods from `ClientBuilderInterface`, use `ClientBuilderInterface::setTransportFactory()` instead: 
   - `ClientBuilderInterface::setUriFactory()`
   - `ClientBuilderInterface::setMessageFactory()`
   - `ClientBuilderInterface::setTransport()`
   - `ClientBuilderInterface::setHttpClient()`
   - `ClientBuilderInterface::addHttpClientPlugin()`
   - `ClientBuilderInterface::removeHttpClientPlugin()`
```